CONTRIBUTING.md

```diff
@@ -0,0 +1,151 @@
+
+## Requesting Features
+
+Feature requests are welcome but take a moment to think if your idea follows the [roadmap](https://forum.minetest.net/viewtopic.php?t=9177) in some way and fits the whole picture of the project. You should provide a clear explanation with as much detail as possible.
+
+## Translating
+
+Translations of Minetest are performed using a web interface called [Weblate](https://hosted.weblate.org/projects/minetest/minetest/).
+
+Current languages can be found in the po/ directory, create an issue if you'd like to translate into a language that isn't there.
+
+## Other ways of contributing
+
+### Creating Content
+
+You can create your own [mods](http://www.minetest.net/customize/#mods), [texture packs](http://www.minetest.net/customize/#texture-packs) or [subgames](http://www.minetest.net/customize/#subgames) and submit them to the [community](https://forum.minetest.net/).
+
+**Resources:**
+
+- [Mods](https://forum.minetest.net/viewforum.php?f=46) / [Texture packs](https://forum.minetest.net/viewforum.php?f=4) / [Subgames](https://forum.minetest.net/viewforum.php?f=48) forum
+- [Online Modding Tutorial Book](http://rubenwardy.com/minetest_modding_book/)
+- [Find Ideas](https://forum.minetest.net/viewtopic.php?f=9&t=2434)
+
+### Donate
+
+If you'd like to monetarily support Minetest, you can find donation methods on [the website](http://www.minetest.net/development/#donate).
+
+# Maintainer Notes
+
+*This is a concise version of the Rules & Guidelines on the [developer wiki](http://dev.minetest.net/Category:Rules_and_Guidelines).*
+
+These notes are for those who have push access Minetest (core developers / maintainers).
+
+## Subsystems
+
+Minetest is split into *subsystems* based on general parts of the code. Each subsystem is allocated to a certain core developer, who becomes the *subsystem maintainer*. They are responsible for handling pull requests and ensuring the subsystem works. They can give their responsibilities to someone else if they see fit.
+
+Subsystem maintainers can be proposed by anyone and they are not-so-officially majority voted.
+
+Developer            |Subsystem
+---------------------|---------------------
+celeron55            | Maintainer conflicts
+sapier               | @Server/Client/Env.
+kahrl                | @Startup/Config/Util
+ShadowNinja	         | @Script API
+hmmmm/kwolekr        | @Mapgen
+celeron55 [HALF-OPEN]| @Client/Audiovisuals
+nrzkt/nerzhul        | @Network
+[OPEN]               | @Build
+ShadowNinja          | @Documentation
+[OPEN]               | @Translation
+[OPEN]               | masterserver
+
+## Reviewing Pull Requests
+
+Pull requests should be reviewed and, if appropriate, checked if they achieve their intended purpose. You can show that you are in the process of, or will review the pull request by commenting "Looks good" or something similar.
```

Comment:
> Right now, "Looks good" is seen as equivalent to +1 and I use it when I haven't tested a patch, but think it needs testing. Other devs might use it at other occasions, but generally I haven't heard anybody complain when a PR got merged with "Looks good" approvals.

The way you use "looks good" seems to be the same way other developers use it so I've updated this PR.